README.md

```diff
@@ -0,0 +1,32 @@
+# Github Hardware Tests
+This package provides a python script periodically for periodically checking a github repo and running tests for PullRequests using [industrial_ci](https://github.com/ros-industrial/industrial_ci).
+
+To avoid running arbitrary PRs on the machine only PullRequests containing the string "- [ ] Perform hardware tests" and with commits from allowed users are checked. The start and finish of the test are reported with a comment on the issue.
+
+## Prequisites
+In order to run the script you need:
+- Docker installed on the machine
+- A personal token with "public_repo" scope
+- `industrial_ci` installed locally and sourced (see instructions [here](https://github.com/ros-industrial/industrial_ci/blob/master/doc/index.rst#simplest-way-to-run-locally))
+## Usage
+```
+rosrun pilz_github_ci_runner test_repository.py "PilzDE/psen_scan_v2" rfeistenauer agutenkunst --token=INSERT_ACTUAL_TOKEN --apt_proxy=http://172.20.20.104:3142 --docker_opts="-v /usr/local/share/ca-certificates:/usr/local/share/ca-certificates:ro --env HOST_IP=192.168.0.122 --env SENSOR_IP=192.168.0.100" --setup_cmd="usbrelay 1_1=0; sleep 2; usbrelay 1_1=1" --cleanup_cmd="usbrelay 1_1=0"
```

Comment:
Since this is psen_scan_v2 specific, would it be the right place for adding `--cmake-args="DENABLE_HARDWARE_TESTING"`?

Reply:
Added!